MSAL/src/requests/MSALSilentRequest.m

```diff
@@ -209,10 +336,11 @@ - (void)acquireTokenWithRefreshToken:(MSIDRefreshToken *)refreshToken
 
 - (MSIDTokenRequest *)tokenRequest
 {
+    MSIDRefreshToken *refreshToken = [_familyId isEqualToString:@"1"] ? _familyRefreshToken : _refreshToken;
```

Comment:
nit: you can just set _familyRefreshToken to nil after you're done with tryFRT, this check then becomes _familyRefreshToken ? _familyRefreshToken : _refreshToken.

Reply:
Sure